<launch>
 <arg name="rplidar_frame" default="laser" />
 <arg name="scan_topic"    default="/scan" />

 <node pkg="rplidar_python" type="rplidar_node.py" name="rplidar" output="screen" respawn="true">
  <param name="scan_topic" value="$(arg scan_topic)"/>
  <param name="rplidar_frame" value="$(arg rplidar_frame)" />
  <param name="range_min" value="0.15"/>
  <param name="range_max" value="6.0"/>
  <param name="rplidar_port_name" value="CP2102 USB to UART Bridge Controller"/>
 </node>
</launch>

